epub33/core/index.html

```diff
@@ -8711,9 +8711,49 @@ <h2>Security and Privacy</h2>
 		<section id="app-overview-unsupported" class="appendix">
 			<h2>Unsupported Features</h2>
 
-			<p>This specification contains certain features the Working Group no longer recommends for use or that are
-				only retained for legacy support reasons. This section defines the meanings of the designations attached
-				to these features and their support expectations.</p>
+			<p>This specification contains certain features that are not yet fully supported in Reading Systems, that
+				the Working Group no longer recommends for use, or that are only retained for interoperability with EPUB
+				2 Reading Systems. This section defines the meanings of the designations attached to these features and
+				their support expectations.</p>
+
+			<section id="risky">
+				<h3>Risky Features</h3>
+
+				<p>A <strong>risky</strong> feature is a feature introduced prior to EPUB 3.3 but that the Working Group
+					has not been able to establish is implemented in two or more Reading Systems as required by <a
+						href="https://www.w3.org/Consortium/Process/">W3C Process</a>.</p>
+
```

Comment:
I would not refer to the W3C Process here.

The fact of "two or more Reading systems" is _not_, formally, part of the W3C Process. There is a separate section in the process on ["implementation experience"](https://www.w3.org/2021/Process-20211102/#adequate-implementation) but each WG has to decide how the general terms are mapped against what is described in the process.

I would say something more general

> has not been able to establish enough implementation experience.

And possibly point to the process document section above.

Reply:
You're the expert!